openapi-v1.yaml

```diff
@@ -2059,6 +2059,13 @@ definitions:
         type: string
         description: Docker image name to use for UDF
         x-omitempty: true
+      runtime_profile:
+        type: string
+        description: >
+          The runtime profile to use for the UDF execution. Profiles define resource limits for memory and cpus.
+          If this is empty, then the UDF will execute in the standard environment of the tiledb-cloud provider.
+        x-omitempty: true
+        example: standard
```

Comment:
The options should be listed out in an enum.

Reply:
What about enterprise users? Shouldn't they be able to define their own names for profiles?

Reply:
I'm with @anastasop here that this should just be a string, and that the values should be specific to the provider. We don't want to lock in users who are hosting their own TileDB Cloud instance into only "standard" and "large". And we don't want to have to push out an API change and a client release and a server release and and and just to add a new resource profile.